dynet/rnn.h

```diff
@@ -166,10 +166,19 @@ struct RNNBuilder {
   }
 
   /**
-   *
-   * \brief Set Dropout
-   *
    * \param d Dropout rate
+   * \details The dropout implemented here is the variational dropout with tied weights introduced in [Gal, 2016](http://papers.nips.cc/paper/6241-a-theoretically-grounded-application-of-dropout-in-recurrent-neural-networks)
```

Comment:
I'm assuming you mean tied masks and not tied weights here?

Reply:
You are right, there is no tied / un-tied variants for SimpleRNN, as there's single gate in SimpleRNN.